Extra/Settings/SettingGui.js

```diff
@@ -105,6 +105,80 @@ class SettingGui {
 
         this.addToggle("Map enabled", "showMap", this.currentSettings.showMap)
 
+        // Idk a good way to put this into the menu but heres smth
+        //this.addSidebarElement(new SoopyTextElement().setText("§0" + "Settings Profile Selector ->").setMaxTextScale(2), 0.1, 0.75)
+        this.profileDropdown = this.addDropdown("Profile Selection",
+            this.currentSettings.getProfiles(),
+            "activeProfile",
+            "Profile 1")[0]
+
+        this.addGear(() => {
+            return true
+        }, (elm) => {
+            let textBox = new TextBox().setText(this.currentSettings["profileName"] ?? this.currentSettings.activeProfile)
+            textBox.text.addEvent(new SoopyContentChangeEvent().setHandler((val, prev, cancelFun) => {
+                this.changed("profileName", val)
+                this.profileDropdown.setOptions(this.currentSettings.getProfiles())
+                this.profileDropdown.setSelectedOption(this.currentSettings.activeProfile)
+            }))
+
+            elm.addSidebarElement(textBox, 0.55, 0.35, 0.05)
+            elm.addSidebarElement(new SoopyTextElement().setText("§0Profile Name").setMaxTextScale(2), 0.1, 0.35)
+
+            elm.addSidebarElement(new ButtonWithArrow().setText("&0Export Profile").addEvent(new SoopyMouseClickEvent().setHandler(() => {
+                ChatLib.command(`ct copy ${JSON.stringify(this.currentSettings.profiles[this.currentSettings.activeProfile])}`, true)
```

Comment:
btw use 
```js
Java.type("net.minecraft.client.gui.GuiScreen")[m.setClipboardString](wayStr)
```
instead of /ct copy
(I could see using that command causing issues in the future)